422 - Excel Your InputStream was neither an OLE2 stream, nor an OOXML stream

Using your test from the readme for excel throwing an error. Other examples working fine :

excel plugin currently read first sheet.

```
SELECT * FROM
storage.excel."https://raw.githubusercontent.com/snowlift/trino-storage/master/src/test/resources/example-data/sample.xlsx";

```
Stack:
```
2023-08-08 22:15:22 2023-08-08T20:15:22.945Z    ERROR   Query-20230808_201522_00026_aszsk-340   org.ebyhr.trino.storage.StorageClient   Failed to get table: excel.https://raw.githubusercontent.com/snowlift/trino-storage/master/src/test/resources/example-data/sample.xlsx
2023-08-08 22:15:22 java.io.UncheckedIOException: java.io.IOException: Your InputStream was neither an OLE2 stream, nor an OOXML stream or you haven't provide the poi-ooxml*.jar in the classpath/modulepath - FileMagic: OOXML, having providers: [org.apache.poi.hssf.usermodel.HSSFWorkbookFactory@460d8f5b]
2023-08-08 22:15:22     at org.ebyhr.trino.storage.operator.ExcelPlugin.getFields(ExcelPlugin.java:61)
2023-08-08 22:15:22     at org.ebyhr.trino.storage.StorageClient.getTable(StorageClient.java:84)
2023-08-08 22:15:22     at org.ebyhr.trino.storage.StorageMetadata.getTableHandle(StorageMetadata.java:74)
2023-08-08 22:15:22     at org.ebyhr.trino.storage.StorageMetadata.getTableHandle(StorageMetadata.java:45)
2023-08-08 22:15:22     at io.trino.spi.connector.ConnectorMetadata.getTableHandle(ConnectorMetadata.java:122)
2023-08-08 22:15:22     at io.trino.tracing.TracingConnectorMetadata.getTableHandle(TracingConnectorMetadata.java:142)
2023-08-08 22:15:22     at io.trino.metadata.MetadataManager.lambda$getTableHandle$5(MetadataManager.java:282)
2023-08-08 22:15:22     at java.base/java.util.Optional.flatMap(Optional.java:289)
2023-08-08 22:15:22     at io.trino.metadata.MetadataManager.getTableHandle(MetadataManager.java:276)
2023-08-08 22:15:22     at io.trino.metadata.MetadataManager.getRedirectionAwareTableHandle(MetadataManager.java:1579)
2023-08-08 22:15:22     at io.trino.metadata.MetadataManager.getRedirectionAwareTableHandle(MetadataManager.java:1571)
2023-08-08 22:15:22     at io.trino.tracing.TracingMetadata.getRedirectionAwareTableHandle(TracingMetadata.java:1291)
2023-08-08 22:15:22     at io.trino.sql.analyzer.StatementAnalyzer$Visitor.getTableHandle(StatementAnalyzer.java:5444)
2023-08-08 22:15:22     at io.trino.sql.analyzer.StatementAnalyzer$Visitor.visitTable(StatementAnalyzer.java:2218)
2023-08-08 22:15:22     at io.trino.sql.analyzer.StatementAnalyzer$Visitor.visitTable(StatementAnalyzer.java:488)
2023-08-08 22:15:22     at io.trino.sql.tree.Table.accept(Table.java:60)
2023-08-08 22:15:22     at io.trino.sql.tree.AstVisitor.process(AstVisitor.java:27)
2023-08-08 22:15:22     at io.trino.sql.analyzer.StatementAnalyzer$Visitor.process(StatementAnalyzer.java:505)
2023-08-08 22:15:22     at io.trino.sql.analyzer.StatementAnalyzer$Visitor.analyzeFrom(StatementAnalyzer.java:4503)
2023-08-08 22:15:22     at io.trino.sql.analyzer.StatementAnalyzer$Visitor.visitQuerySpecification(StatementAnalyzer.java:2981)
2023-08-08 22:15:22     at io.trino.sql.analyzer.StatementAnalyzer$Visitor.visitQuerySpecification(StatementAnalyzer.java:488)
2023-08-08 22:15:22     at io.trino.sql.tree.QuerySpecification.accept(QuerySpecification.java:155)
2023-08-08 22:15:22     at io.trino.sql.tree.AstVisitor.process(AstVisitor.java:27)
2023-08-08 22:15:22     at io.trino.sql.analyzer.StatementAnalyzer$Visitor.process(StatementAnalyzer.java:505)
2023-08-08 22:15:22     at io.trino.sql.analyzer.StatementAnalyzer$Visitor.process(StatementAnalyzer.java:513)
2023-08-08 22:15:22     at io.trino.sql.analyzer.StatementAnalyzer$Visitor.visitQuery(StatementAnalyzer.java:1501)
2023-08-08 22:15:22     at io.trino.sql.analyzer.StatementAnalyzer$Visitor.visitQuery(StatementAnalyzer.java:488)
2023-08-08 22:15:22     at io.trino.sql.tree.Query.accept(Query.java:107)
2023-08-08 22:15:22     at io.trino.sql.tree.AstVisitor.process(AstVisitor.java:27)
2023-08-08 22:15:22     at io.trino.sql.analyzer.StatementAnalyzer$Visitor.process(StatementAnalyzer.java:505)
2023-08-08 22:15:22     at io.trino.sql.analyzer.StatementAnalyzer.analyze(StatementAnalyzer.java:467)
2023-08-08 22:15:22     at io.trino.sql.analyzer.Analyzer.analyze(Analyzer.java:97)
2023-08-08 22:15:22     at io.trino.sql.analyzer.Analyzer.analyze(Analyzer.java:86)
2023-08-08 22:15:22     at io.trino.execution.SqlQueryExecution.analyze(SqlQueryExecution.java:270)
2023-08-08 22:15:22     at io.trino.execution.SqlQueryExecution.<init>(SqlQueryExecution.java:205)
2023-08-08 22:15:22     at io.trino.execution.SqlQueryExecution$SqlQueryExecutionFactory.createQueryExecution(SqlQueryExecution.java:844)
2023-08-08 22:15:22     at io.trino.dispatcher.LocalDispatchQueryFactory.lambda$createDispatchQuery$0(LocalDispatchQueryFactory.java:153)
2023-08-08 22:15:22     at io.trino.$gen.Trino_422____20230808_200218_2.call(Unknown Source)
2023-08-08 22:15:22     at com.google.common.util.concurrent.TrustedListenableFutureTask$TrustedFutureInterruptibleTask.runInterruptibly(TrustedListenableFutureTask.java:131)
2023-08-08 22:15:22     at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:75)
2023-08-08 22:15:22     at com.google.common.util.concurrent.TrustedListenableFutureTask.run(TrustedListenableFutureTask.java:82)
2023-08-08 22:15:22     at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
2023-08-08 22:15:22     at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
2023-08-08 22:15:22     at java.base/java.lang.Thread.run(Thread.java:833)
2023-08-08 22:15:22 Caused by: java.io.IOException: Your InputStream was neither an OLE2 stream, nor an OOXML stream or you haven't provide the poi-ooxml*.jar in the classpath/modulepath - FileMagic: OOXML, having providers: [org.apache.poi.hssf.usermodel.HSSFWorkbookFactory@460d8f5b]
2023-08-08 22:15:22     at org.apache.poi.ss.usermodel.WorkbookFactory.wp(WorkbookFactory.java:334)
2023-08-08 22:15:22     at org.apache.poi.ss.usermodel.WorkbookFactory.create(WorkbookFactory.java:224)
2023-08-08 22:15:22     at org.apache.poi.ss.usermodel.WorkbookFactory.create(WorkbookFactory.java:185)
2023-08-08 22:15:22     at org.ebyhr.trino.storage.operator.ExcelPlugin.getFields(ExcelPlugin.java:49)
2023-08-08 22:15:22     ... 43 more
2023-08-08 22:15:22 
2023-08-08 22:15:22
```

--- a/pom.xml
+++ b/pom.xml
@@ -237,6 +237,13 @@
         </dependency>
 
         <dependency>
+            <groupId>org.apache.poi</groupId>
+            <artifactId>poi-ooxml</artifactId>
+            <version>5.2.3</version>
+            <scope>runtime</scope>
+        </dependency>
+
+        <dependency>
             <groupId>io.airlift</groupId>
             <artifactId>units</artifactId>
             <version>1.10</version>
@@ -345,13 +352,6 @@
             <groupId>net.java.dev.jna</groupId>
             <artifactId>jna</artifactId>
             <version>5.13.0</version>
-            <scope>test</scope>
-        </dependency>
-
-        <dependency>
-            <groupId>org.apache.poi</groupId>
-            <artifactId>poi-ooxml</artifactId>
-            <version>5.2.3</version>
             <scope>test</scope>
         </dependency>
 

